concourse/scripts/s3init.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Then we either:

 * Create a new container including bash.
 * do not use `[[`

I think all of that can be done in the other story about refactor s3init.sh, as it was played later. If we block this PR longer, we block all the other cards.